### PR TITLE
Add named-capturing group

### LINKED ISSUE #67: Named-capturing group doesn't seem to be supported
The `(?<name>...)` syntax is not supported

### PR DESCRIPTION
**Pull request by tkzwyk**
Fixed #67

### CODE REVIEW

**Comment by tkzwyk** [comment]
Hi @lanwen , do you know why coverage decreasing?

**Comment by lanwen** [comment]
Seems that it was not collected at all for some reason

**Comment by tkzwyk** [comment]
Hi @lanwen , thank you for reply. I updated `coveralls-maven-plugin` version to 4.3.0 (latest), so the coverage increased. 
Could you please review this pr?

File: src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java [hunk 1]
```diff
@@ -133,4 +133,20 @@ public void captureAfterNewLineHasGroupNumberOne() throws Exception {
 
         assertEquals(some, expression.getText(line_break + some + text, 1));
     }
+
+    @Test
+    public void captureAfterNewLineHasANamedGroup() {
+
+        final String line_break = "\n";
```
Comment (lanwen) [inline src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java]: Plz camelCase only

File: src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java [hunk 2]
```diff
@@ -133,4 +133,20 @@ public void captureAfterNewLineHasGroupNumberOne() throws Exception {
...
+        final String some = "some";
+        final String text = " text";
+        final String captureName = "name";
+        final VerbalExpression expression = VerbalExpression.regex().
+                lineBreak()
+                .capture(captureName).find(some).endCapture().then(text)
+                .build();
+
+        assertEquals(some,
```
Comment (lanwen) [inline src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java]: Plz use assertThat instead

File: src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java [hunk 3]
```diff
@@ -563,7 +655,7 @@ public void zeroOreMoreSameAsAtLeast0() throws Exception {
         assertThat(regexWithOneOrMore, matchesTo(empty));
         assertThat(regexWithOneOrMore, matchesExactly(empty));
     }
-
```
Comment (lanwen) [inline src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java]: Please avoid such changes

File: src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java (hunk 1 above)
Comment (tkzwyk) [reply src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java]: Thanks for advise. I fixed by d3a0488

File: src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java (hunk 2 above)
Comment (tkzwyk) [reply src/test/java/ru/lanwen/verbalregex/RealWorldUnitTest.java]: Thanks for advise. I fixed by edb2a18

File: src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java (hunk 3 above)
Comment (tkzwyk) [reply src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java]: Thanks for advise. Should I undo this modification

**Comment by tkzwyk** [comment]
Thank you for merging

File: src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java (hunk 3 above)
Comment (tkzwyk) [reply src/test/java/ru/lanwen/verbalregex/BasicFunctionalityUnitTest.java]: I will be careful about that from next time. Thanks

**Pull request merged**